### ISSUE TITLE
chifra scrape crash

### ISSUE THREAD

**Issue by marsu-dev** [TrueBlocks/trueblocks-core#2474]
Version Chifra  `v0.41.0` from trueblocks-docker

Steps to reproduce
- chifra init --all
- chifra scrape

Once the tip reached crash append

```
INFO[22-10|03:38:23.749] No new blocks at block 15801004 (2 away from head)
INFO[22-10|03:38:24.000] Sleeping for 14 seconds - 2 away from head.
panic: runtime error: makeslice: cap out of range

goroutine 1 [running]:
github.com/TrueBlocks/trueblocks-core/src/apps/chifra/internal/scrape.(*ScrapeOptions).HandleScrape(0x11d2100)
        /root/trueblocks-core/src/apps/chifra/internal/scrape/handle_scrape.go:81 +0x3b0
github.com/TrueBlocks/trueblocks-core/src/apps/chifra/internal/scrape.(*ScrapeOptions).ScrapeInternal(0x11d2100)
        /root/trueblocks-core/src/apps/chifra/internal/scrape/output.go:52 +0x7f
github.com/TrueBlocks/trueblocks-core/src/apps/chifra/internal/scrape.RunScrape(0xb04400?, {0x126c1b8?, 0xbbd410?, 0x8?})
        /root/trueblocks-core/src/apps/chifra/internal/scrape/output.go:27 +0x2c
github.com/TrueBlocks/trueblocks-core/src/apps/chifra/pkg/file.RunWithFileSupport.func1(0x11d49e0?, {0x126c1b8?, 0x0, 0x0})
        /root/trueblocks-core/src/apps/chifra/pkg/file/commands.go:110 +0x18c
github.com/spf13/cobra.(*Command).execute(0x11d49e0, {0x126c1b8, 0x0, 0x0})
        /go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:856 +0x67c
github.com/spf13/cobra.(*Command).ExecuteC(0x11d4760)
        /go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:974 +0x3b4
github.com/spf13/cobra.(*Command).Execute(...)
        /go/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:902
github.com/TrueBlocks/trueblocks-core/src/apps/chifra/cmd.Execute()
        /root/trueblocks-core/src/apps/chifra/cmd/root.go:28 +0x25
main.main()
        /root/trueblocks-core/src/apps/chifra/main.go:20 +0x25
  ```

**Comment by marsu-dev** [comment]
Restarting chifra scrape went well

**Comment by tjayrush** [comment]
Thanks for your PR. I made some comments.

**Comment by tjayrush** [comment]
I'm pretty sure we've fixed this in the next release (v0.44.0) which we're going to push later this week.

If you get a chance, once we make the announcement (in discord announcements channel), check it out.

I'm going to close this, but if you continue to see the problem, please feel free to re-open.
